### PR TITLE
Add a bunch of const keywords to allow usage of LogError/LogWarning/LogNote in const functions without const_cast

### PR DESCRIPTION
**Pull request by MaksymKapelianovych**

### CODE REVIEW

**Comment by MothDoctor** [comment]
Hi, please grab the latest changes as this change has merge conflicts with just accepted Flow AddOns.

**Comment by MaksymKapelianovych** [comment]
Updated!

File: Source/Flow/Private/Nodes/FlowNodeBase.cpp [hunk 1]
```diff
@@ -612,12 +612,7 @@ void UFlowNodeBase::LogError(FString Message, const EFlowOnScreenMessageType OnS
 #endif
 }
 
-void UFlowNodeBase::LogErrorConst(FString Message, const EFlowOnScreenMessageType OnScreenMessageType) const
```
Comment (MothDoctor) [inline Source/Flow/Private/Nodes/FlowNodeBase.cpp:635]: I would keep separate const and const-version as it sometimes nodes need to modify some internal properties

File: Source/Flow/Private/Nodes/FlowNodeBase.cpp [hunk 2]
```diff
@@ -633,12 +628,7 @@ void UFlowNodeBase::LogWarning(FString Message)
 #endif
 }
 
-void UFlowNodeBase::LogWarningConst(FString Message) const
```
Comment (MothDoctor) [inline Source/Flow/Private/Nodes/FlowNodeBase.cpp:656]: I would keep separate const and const-version as it sometimes nodes need to modify some internal properties

File: Source/Flow/Private/Nodes/FlowNodeBase.cpp [hunk 3]
```diff
@@ -633,12 +628,7 @@ void UFlowNodeBase::LogWarning(FString Message)
...
-{
-	const_cast<UFlowNodeBase*>(this)->LogWarning(Message);
-}
-
-void UFlowNodeBase::LogNote(FString Message)
+void UFlowNodeBase::LogNote(FString Message) const
```
Comment (MothDoctor) [inline Source/Flow/Private/Nodes/FlowNodeBase.cpp:651]: I would keep separate const and const-version as it sometimes nodes need to modify some internal properties

**Comment by MothDoctor** [comment]
One more merge fix will be needed here. I will pull as the next PR after merge fix and applying my comments :)

File: Source/Flow/Private/Nodes/FlowNodeBase.cpp (hunk 1 above)
Comment (MaksymKapelianovych) [reply Source/Flow/Private/Nodes/FlowNodeBase.cpp:635]: I am afraid I do not understand your point here... LogError and LogErrorConst can be used in both const and non-const functions (if this PR will be merged), they itself do not modify any properties. Could you please give an example where I can use LogErrorConst, but not LogError?

Comment (MothDoctor) [reply Source/Flow/Private/Nodes/FlowNodeBase.cpp:635]: OK :)

**Pull request merged**